extensions/wikia/PortableInfobox/templates/PortableInfoboxItemHeaderMobile.mustache

```diff
@@ -1,3 +1,3 @@
 <div class="pi-header-wrapper">
-    <h2 class="pi-item pi-header pi-secondary-font pi-item-spacing pi-secondary-background"{{#inlineStyles}} style="{{inlineStyles}}"{{/inlineStyles}}>{{{value}}}</h2>
+    <h2 class="pi-item pi-header pi-secondary-font pi-item-spacing pi-secondary-background wds-midlight-navy"{{#inlineStyles}} style="{{inlineStyles}}"{{/inlineStyles}}>{{{value}}}</h2>
```

Comment:
@bkoval IMO we should extend %wds-midlight-navy on scss side in mobile-wiki as with the rest of styles in infoboxes